core/src/com/unciv/logic/battle/BattleDamage.kt

```diff
@@ -45,18 +45,13 @@ object BattleDamage {
         val civInfo = combatant.getCivInfo()
         if (combatant is MapUnitCombatant) {
             for (BDM in getBattleDamageModifiersOfUnit(combatant.unit)) {
-                if (BDM.vs == enemy.getUnitType().toString())
-                    addToModifiers(BDM)
-                if (BDM.vs == "wounded units" && enemy is MapUnitCombatant && enemy.getHealth() < 100)
-                    addToModifiers(BDM)
-                if (BDM.vs == "land units" && enemy.getUnitType().isLandUnit())
-                    addToModifiers(BDM)
-                if (BDM.vs == "water units" && enemy.getUnitType().isWaterUnit())
-                    addToModifiers(BDM)
-                if (BDM.vs == "air units" && enemy.getUnitType().isAirUnit())
+                if (enemy.matchesCategory(BDM.vs)) {
                     addToModifiers(BDM)
+                }
+
             }
 
+            // As of 3.11.1 This is to be deprecated and converted to "Bonus vs x y%" - keeping it here so that mods with this can still work for now
```

Comment:
Au Contrair, I want to convert all the Bonus vs x y% TO +[]% Strength vs [] ;)

Better because of:
- Unified unique handling
- Less regex use
- No need for custom translation

Reply:
Not that I think a *comment* is a good enough reason to avoid merging this, just FYI ;)